dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/visitor/generator/ImportGenerator.java

```diff
@@ -50,10 +50,13 @@ public void startGeneration(int section, JavaCodeWriter writer)
 					String directiveValue = attribute.getNodeValue();
 					if (directiveName.equals("import")) {
 						StringTokenizer tokenizer = new StringTokenizer(directiveValue, ",");
+						String STATIC_IMPORT_PREPPEND = "static ";
 						writeDebugStartBegin(writer);
 						while (tokenizer.hasMoreTokens()) {
 							String singleImport = ((String) tokenizer.nextToken()).trim();
-							if(singleImport.endsWith(".*")){
+							if (singleImport.startsWith(STATIC_IMPORT_PREPPEND)) {
+							        writer.println("importStaticList.add(\"" + singleImport.substring(STATIC_IMPORT_PREPPEND.length()) + "\");");
+							} else if(singleImport.endsWith(".*")){
```

Comment:
If there's more than one space between static and the package, could that could break things? 

i.e 'static   Math.PI`

Reply:
It did not, nice catch! Just pushed out a fix for it.